packages/app/src/front/views/ListView.tsx

```diff
@@ -105,12 +108,14 @@ export default function ListView() {
     }
   }
 
+  const filteredMessages = messages.filter((message) => selectedLevels.includes(message.data.level))
```

Comment:
I think we should memoize this value because if we have thousands of logs it could create performance issues